Darkly/LDClient.h

```diff
@@ -39,7 +40,27 @@
  * @param default   Default value for feature flag
  * @return the feature flag value
  */
-- (BOOL)toggle:(NSString *)featureName default:(BOOL)defaultValue;
+- (BOOL)toggle:(NSString *)featureKey defaultValue:(BOOL)defaultValue;
+/**
+ * Retrieve a feature flag value. If the configuration for this feature
+ * flag is retrieved from the server that value is returned, otherwise
+ * the defaultValue is returned.
+ *
+ * @param featureName   Name of feature flag
+ * @param default   Default value for feature flag
+ * @return the feature flag value
+ */
+- (NSNumber*)numberVariation:(NSString *)featureKey defaultValue:(NSNumber*)defaultValue;
+/**
+ * Retrieve a feature flag value. If the configuration for this feature
+ * flag is retrieved from the server that value is returned, otherwise
+ * the defaultValue is returned.
+ *
+ * @param featureName   Name of feature flag
+ * @param default   Default value for feature flag
+ * @return the feature flag value
+ */
+- (NSString*)stringVariation:(NSString *)featureKey defaultValue:(NSString*)defaultValue;
```

Comment:
In Java, we also have a `jsonVariation` method, as LD now supports json objects and arrays:

https://github.com/launchdarkly/java-client/blob/v2/src/main/java/com/launchdarkly/client/LDClient.java#L288

What's the objective C equivalent? Two methods that return `NSMutableDictionary` (for json objects) or `NSMutableArray` (for json arrays)? Or is there a common AST type we can return as in Java?